game/typechart.py

```diff
@@ -2,6 +2,8 @@
 
 
 class Element(Enum):
+    """Element property declared here"""
```

Comment:
Kind of redundant, I would just say that these are the different options a Pokemon's type can be

Reply:
Changed to `Possible types that Pokemon can be`